src/state/routing/utils.ts

```diff
@@ -71,6 +79,13 @@ export function transformRoutesToTrade<TTradeType extends TradeType>(
       route
         ?.filter((r): r is typeof route[0] & { routev3: NonNullable<typeof route[0]['routev3']> } => r.routev3 !== null)
         .map(({ routev3, inputAmount, outputAmount }) => ({ routev3, inputAmount, outputAmount })) ?? [],
+    mixedRoutes:
```

Comment:
Where is this used?

Reply:
used to build the `InterfaceTrade` which is returned by `useRoutingAPITrade()` and then in `useBestTrade()`